README.md

```diff
@@ -299,6 +316,36 @@ Since npm does it, we should do it:
 - Scripts like this `"eslint": "eslint"` used to be common to make dependencies executable with `npm run`. Since [`npx`](https://www.npmjs.com/package/npx) (and with yarn of course) this is obsolete. There's a lint rule in the CLI for this.
 - scripts that start with `pre*` or `post*` should only be used as hooks for built-in commands or other scripts
 
+## scriptlint as a module dependency
+
+You can install scriptlint as a dependency and call it as a function like this:
+
+```
```

Comment:
We could make the README a bit prettier with syntax highlighting. 

What do you think?

```suggestion
```js
```